sound/soc/sof/ipc4-topology.h

```diff
@@ -42,6 +42,16 @@
 
 #define ALH_MAX_NUMBER_OF_GTW   16
 
+/*
+ * The base of multi-gateways. Multi-gateways addressing starts from
+ * ALH_MULTI_GTW_BASE and there are ALH_MULTI_GTW_COUNT multi-sources
+ * and ALH_MULTI_GTW_COUNT multi-sinks available.
+ * Addressing is continuous from ALH_MULTI_GTW_BASE to
+ * ALH_MULTI_GTW_BASE + ALH_MULTI_GTW_COUNT - 1.
+ */
+#define ALH_MULTI_GTW_BASE	0x50
+#define ALH_MULTI_GTW_COUNT	9
```

Comment:
we should explain what this 9 means?

Reply:
> we should explain what this 9 means?

That was from [#define SDW_MAX_GROUPS 9](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/intel/boards/sof_sdw_common.h#L24). copying the same comment from there.

Reply:
Ah no, the comments on SDW_MAX_GROUPS was purely based on the number of links we manage.  This could change at any time if someone comes up with a configuration with more links.

The multi-gateway stuff is more of an ABI, so that has to be more conservative and with more future-proofing.

Those are different things.

Reply:
@plbossart I checked cavs_fw code. It is defined in hw_defs.h for each platform. The value of all existing platforms are 8. But there is no comment for it.